website/docs/d/users.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "gitlab"
+page_title: "GitLab: gitlab_users"
+sidebar_current: "docs-gitlab-data-source-users"
+description: |-
+  Looks up gitlab users
+---
+
+# gitlab\_users
+
+Provides details about a list of users in the gitlab provider. The results include id, username, email, name and more about the requested users. Users can also be sorted and filtered using several options.
+
+## Example Usage
+
+```hcl
+data "gitlab_users" "example" {
+  sort = "desc"
+  order_by = "name"
+  created_before = "2019-01-01"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `search` - (Optional) Search users by username, name or email.
+
+* `active` - (Optional) Filter users that are active.
+
+* `blocked` - (Optional) Filter users that are blocked.
+
+* `order_by` - (Optional) Order the users' list by `id`, `name`, `username`, `created_at` or `updated_at`. (Requires administrator privileges)
+
+* `sort` - (Optional) Sort users' list in asc or desc order. (Requires administrator privileges)
+
+* `identities_extern_uid` - (Optional) Lookup users by external UID. (Requires administrator privileges)
```

Comment:
Maybe rename this to `identities_external_uid` or simply `external_uid`?

Reply:
I set it to `identities_extern_uid` to keep consistency with `identities_provider` which can't be renamed into `provider` (forbidden Terraform keyword). But I don't mine changing it if you think this is clearer/

Reply:
Ok, what about `extern_uid` (to match the API as you suggest) and `extern_provider` (to match the first one).

https://github.com/xanzy/go-gitlab/blob/master/users.go#L88

Reply:
Done.